numpy/lib/_polynomial_impl.py

```diff
@@ -457,7 +457,7 @@ def polyfit(x, y, deg, rcond=None, full=False, w=None, cov=False):
        A summary of the differences can be found in the
        :doc:`transition guide </reference/routines.polynomials>`.
 
-    Fit a polynomial ``p(x) = p[0] * x**deg + ... + p[deg]`` of degree `deg`
+    Fit a polynomial ``f(x) = p[0] * x**deg + ... + p[deg]`` of degree `deg`
```

Comment:
I'd be inclined to just remove `p(x) = `, I don't think replacing `p` by `f` improves it.

Reply:
Alright got it, shall I close the Pr then as no changes are required?

Reply:
I think the suggestion is to remove p(x) so that it reads like this?

```suggestion
    Fit a polynomial ``p[0] * x**deg + ... + p[deg]`` of degree `deg`
```